projects/gloo/pkg/utils/validation/proxy_validation.go

```diff
@@ -250,6 +258,14 @@ func GetTcpListenerErr(tcpListener *validation.TcpListenerReport) []error {
 	return errs
 }
 
+func GetTcpListenerWarning(tcpListener *validation.TcpListenerReport) []string {
+	var warnings []string
```

Comment:
```suggestion
	warnings := make([]string, len(httpListener.GetWarnings())
```

Reply:
Above should be `tcpListener.GetWarnings`, not `httpListener`